gradle/libs.versions.toml

```diff
@@ -216,21 +216,19 @@ wiremockJunit5 = { module = "ru.lanwen.wiremock:wiremock-junit5", version.ref =
 # plugin dependencies
 plugin-errorprone = { module = "net.ltgt.errorprone:net.ltgt.errorprone.gradle.plugin", version = "4.3.0" }
 plugin-license = { module = "gradle.plugin.com.hierynomus.gradle.plugins:license-gradle-plugin", version = "0.16.1" }
-plugin-nebulaRelease = { module = "com.netflix.nebula:nebula-release-plugin", version = "18.0.8" }
-plugin-nebulaPublishing = { module = "com.netflix.nebula:nebula-publishing-plugin", version = "20.3.0" }
-plugin-nebulaProject = { module = "com.netflix.nebula:nebula-project-plugin", version = "10.1.5" }
-plugin-nebulaInfo = { module = "com.netflix.nebula:gradle-info-plugin", version = "12.1.6" }
+plugin-nebulaRelease = { module = "com.netflix.nebula:nebula-release-plugin", version = "21.0.0" }
+plugin-nebulaPublishing = { module = "com.netflix.nebula:nebula-publishing-plugin", version = "22.0.0" }
+plugin-nebulaProject = { module = "com.netflix.nebula:nebula-project-plugin", version = "12.0.0" }
+plugin-nebulaInfo = { module = "com.netflix.nebula:gradle-info-plugin", version = "15.0.0" }
```

Comment:
I'm thinking if we should delete this. IIRC this line wasn't there in the past, we added because there was a bug in the info plugin which was fixed and released but the `nebula-publishing-plugin` and `nebula-project-plugin` were not upgraded immediately and we needed to "force" a newer version of the info plugin this way.

On the other hand, it seems these plugins still disagree the version of info:
```
nebula-publishing-plugin -> gradle-info-plugin:16.0.0
nebula-project-plugin    -> gradle-info-plugin:14.0.0
```
which should resolve to `16.0.0` but maybe it indicates that we should keep forcing the version.

So I think we should either remove this line or upgrade it to `16.0.0`.